dashboard/config/initializers/new_framework_defaults_7_0.rb

```diff
@@ -11,10 +11,10 @@
 
 # `button_to` view helper will render `<button>` element, regardless of whether
 # or not the content is passed as the first argument or as a block.
-# Rails.application.config.action_view.button_to_generates_button_tag = true
+#Rails.application.config.action_view.button_to_generates_button_tag = true
 
 # `stylesheet_link_tag` view helper will not render the media attribute by default.
-# Rails.application.config.action_view.apply_stylesheet_media_default = false
+#Rails.application.config.action_view.apply_stylesheet_media_default = false
```

Comment:
Hah, so I did. Mostly because that's how the auto-comment functionality of my editor works, but also I do think it looks better when prose comments have spaces between them and the comment characters, but commented-out code doesn't.